docker-compose.yml

```diff
@@ -24,5 +27,32 @@ services:
       - .parking-service.env
     volumes:
     - images:/cowbot-images
-    #ports:
-    #  - '3000:3000'
+
+  skill-search:
+    build:
+      context: ./skill-search
+    restart: unless-stopped
+    env_file:
+      .skill-search.env
+    volumes:
+      - ./skill-search/data/${SQLITE_DATABASE}:/app/data/${SQLITE_DATABASE}:ro
```

Comment:
Is there any reason you wanted this parameterized?  Is there any harm in hard-coding a default?

Reply:
I was allowing the sqlite database file to be changed based on the environment the app is running. (e.g. - skill-search-development.db,  skill-search-production.db). I can definitely hard-code it for the sake of simplicity.